packages/sign-client/test/sdk/auth.spec.ts

```diff
@@ -141,6 +143,8 @@ describe("Authenticated Sessions", () => {
     expect(wallet.auth.requests.getAll().length).to.eq(0);
     expect(dapp.proposal.getAll().length).to.eq(0);
     expect(dapp.auth.requests.getAll().length).to.eq(0);
+
+    await deleteClients({ A: dapp, B: wallet });
```

Comment:
Small nit: I think you might be able to use `afterAll`

Reply:
lets leave for another PR